core/templates/dev/head/components/forms/schema-viewers/schema-based-list-viewer.directive.ts

```diff
@@ -16,6 +16,7 @@
  * @fileoverview Directive for a schema-based viewer for lists.
  */
 
+require('components/forms/schema-viewers/schema-based-viewer.directive.ts');
```

Comment:
Should there be empty line after this line? Because `schema-based-viewer` is used in the HTML?